### PR TITLE
[task] Add a link on the talk ID to open it in a new window

### PR DESCRIPTION
**Pull request by aheritier**
Useful to easily verify something in the talk description or about the scheduling constraint

### CODE REVIEW

**Comment by aheritier** [comment]
Please @nicmarti 
FYI @fcamblor

File: public/angular/partials/slots.html [hunk 1]
```diff
@@ -17,7 +17,7 @@ <h3>{{title}}</h3>
              drag="talk" dragStyle="columnDrag"
              dropStyle="columnDrop" title="{{talk.state.code}}">
             <span class="proposalTitle">{{talk.title}}</span>
-            <span class="badge badge-secondary">{{talk.id}}</span>
+            <a class="badge badge-secondary" href="/cfpadmin/proposal/{{talk.id}}" target="_blank">{{talk.id}}&nbsp;<i class="fas fa-external-link-alt"></i></a>
```
Comment (nicmarti) [inline public/angular/partials/slots.html:20]: It is not a good pattern to hard-code URLs in template. If possible, can you try to use the @routes ? If it is not possible, just let me know. Thanks

**Review by nicmarti — CHANGES_REQUESTED** [review]

File: public/angular/partials/slots.html (hunk 1 above)
Comment (aheritier) [reply public/angular/partials/slots.html:20]: I agree @nicmarti  but here we are in angular (/public/angular/...) thus out of the scope of Play and I didn't find how to do it. Any idea ?

Comment (nicmarti) [reply public/angular/partials/slots.html:20]: ok for me then... And we cannot use Play JS router

**Pull request merged**